go/noise/noise.go

```diff
@@ -100,4 +118,14 @@ type Noise interface {
 	// satisfies (epsilon,deltaNoise+deltaThreshold)-differential privacy under the
 	// given assumptions of L_0 and L_∞ sensitivities.
 	Threshold(l0Sensitivity int64, lInfSensitivity, epsilon, deltaNoise, deltaThreshold float64) float64
+
+	// ReturnConfidenceIntervalInt64 will return a ConfidenceIntervalInt64 struct with the given confidenceLevel, using the int64 noisedValue,
```

Comment:
Could you reformat the line spacing here? It seems this line has become too long. (Also below)